Week-1/tanmay-sinha/GUIapp.py

```diff
@@ -0,0 +1,124 @@
+import tkinter as tk
+import json
+from app import PhoneDetails
+from tkinter import *
+import time
+with open("phoneurls.json","r") as f:
+    cont = json.load(f)
```

Comment:
what is cont exactly? (rhetorical)
Descriptive variable names can be used. Also for constants, I'd recommend using UPPER_CASE naming style.